shop/src/main/java/ro/msg/learning/exception/mapper/LocationExceptionController.java

```diff
@@ -0,0 +1,16 @@
+package ro.msg.learning.exception.mapper;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import ro.msg.learning.exception.SuitableShippingLocationNotFoundException;
+
+@ControllerAdvice
+public class LocationExceptionController {
```

Comment:
could use one RestControllerAdvice for the whole application (AppExceptionController). Add an exception for stock not found for example

Reply:
Fixed